sdk/keyvault/keyvault-keys/karma.conf.js

```diff
@@ -0,0 +1,128 @@
+      "karma-edge-launcher",
+      "karma-firefox-launcher",
+      "karma-ie-launcher",
+      "karma-env-preprocessor",
+      "karma-coverage",
+      "karma-remap-coverage",
+      "karma-junit-reporter",
+      "karma-json-to-file-reporter",
+      "karma-json-preprocessor"
+    ],
+
+    files: [
+      // polyfill service supporting IE11 missing features
+      // Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys
+      "https://cdn.polyfill.io/v2/polyfill.js?features=Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys|always",
+      "dist-test/index.browser.js",
+      "recordings/browsers/**/*.json"
+    ],
+
+    exclude: [],
+
+    preprocessors: {
+      "**/*.js": ["env"],
+      "dist-test/index.browser.js": ["coverage"],
+      "recordings/browsers/**/*.json": ["json"]
+    },
+
+    envPreprocessor: [
+      "AZURE_CLIENT_ID",
+      "AZURE_CLIENT_SECRET",
+      "AZURE_TENANT_ID",
+      "KEYVAULT_NAME",
+      "TEST_MODE"
+    ],
+
+    reporters: ["mocha", "coverage", "remap-coverage", "junit", "json-to-file"],
+
+    coverageReporter: { type: "in-memory" },
+
+    remapCoverageReporter: {
+      "text-summary": null,
+      html: "./coverage-browser",
+      cobertura: "./coverage-browser/cobertura-coverage.xml"
+    },
+
+    remapOptions: {
+      exclude: /node_modules|tests/g
+    },
+
+    junitReporter: {
+      outputDir: "",
+      outputFile: "test-results.browser.xml",
+      suite: "",
+      useBrowserName: false,
+      nameFormatter: undefined,
+      classNameFormatter: undefined,
+      properties: {}
+    },
+
+    jsonToFileReporter: {
+      filter: function(obj) {
+        if (obj.writeFile) {
+          const fs = require("fs-extra");
+          try {
+            // Stripping away the filename from the file path and retaining the directory structure
+            fs.ensureDirSync(obj.path.substring(0, obj.path.lastIndexOf("/") + 1));
+          } catch (err) {
+            if (err.code !== "EEXIST") throw err;
+          }
+          fs.writeFile(obj.path, JSON.stringify(obj.content, null, " "), (err) => {
+            if (err) {
+              throw err;
+            }
+          });
+        }
+        return false;
+      },
+      outputPath: "."
+    },
+
+    port: 9328,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: false,
+
+    // --no-sandbox allows our tests to run in Linux without having to change the system.
+    // --disable-web-security allows us to authenticate from the browser without having to write tests using interactive auth, which would be far more complex.
+    browsers: ["ChromeHeadlessNoSandbox"],
+    customLaunchers: {
+      ChromeHeadlessNoSandbox: {
+        base: "ChromeHeadless",
+        flags: ["--no-sandbox", "--disable-web-security"]
```

Comment:
Why is this flag `--no-sandbox` required ?

Reply:
I haven't been able to make it work in my linux machine without this.

Reply:
Is this the same situation for storage-queue?

Reply:
This is necessary for me on all libraries that use chrome headless 😅  I believe @daviwil knows an alternative, but I believe it changes the kernel or something.